api/tests/integration/infrastructure/repositories/clea-certified-candidate-repository_test.js

```diff
@@ -0,0 +1,216 @@
+const { expect, databaseBuilder, domainBuilder } = require('../../../test-helper');
+const { CLEA } = require('../../../../lib/domain/models/ComplementaryCertification');
+const cleaCertifiedCandidateRepository = require('../../../../lib/infrastructure/repositories/clea-certified-candidate-repository');
+const { PIX_EMPLOI_CLEA_V3 } = require('../../../../lib/domain/models/Badge').keys;
+
+describe('Integration | Repository | clea-certified-candidate-repository', function () {
+  describe('#getBySessionId', function () {
+    describe('when there is candidates for Clea certification in the session', function () {
+      it('returns the list of clea certified candidates', async function () {
+        // given
+        const sessionId = databaseBuilder.factory.buildSession().id;
+        const userId = databaseBuilder.factory.buildCertificationCandidate({
+          email: 'jean-mi@coco.fr',
+          sessionId,
+        }).userId;
+        const userId2 = databaseBuilder.factory.buildCertificationCandidate({
+          email: 'marie-mi@coco.fr',
+          sessionId,
+        }).userId;
+        const candidateCleaSuccess = {
+          firstName: 'Jean-Mi',
+          lastName: 'Mi',
+          birthdate: '2001-02-07',
+          birthplace: 'Paris',
+          sex: 'M',
+          birthPostalCode: null,
+          birthINSEECode: '75115',
+          birthCountry: 'FRANCE',
+          createdAt: new Date('2020-02-01'),
+          userId,
+        };
+        const candidateCleaSuccess2 = {
+          firstName: 'Marie',
+          lastName: 'Ri',
+          birthdate: '2001-02-04',
+          birthplace: 'Orléans',
+          sex: 'F',
+          birthPostalCode: null,
+          birthINSEECode: '75115',
+          birthCountry: 'FRANCE',
+          createdAt: new Date('2020-02-01'),
+          userId: userId2,
+        };
+
+        databaseBuilder.factory.buildComplementaryCertification({
+          id: 1,
+          key: CLEA,
+        });
```

Comment:
On a ajouté recemment un builder specifique clea
```suggestion
        databaseBuilder.factory.buildComplementaryCertification.clea({
          id: 1,
        });
```